### PR TITLE
Fixes #411: Add glustercli usage in the Quick Start Guide.

### LINKED ISSUE #411: Add glustercli usage to Quick Start Guide

### PR DESCRIPTION
**Pull request by devyanikota**
Signed-off-by: Devyani Kota <devyanikota@gmail.com>

### CODE REVIEW

File: doc/quick-start-user-guide.md [hunk 1]
```diff
@@ -149,15 +157,24 @@ Send the volume create request to create a 2x2 distributed-replicate volume:
 $ curl -X POST http://192.168.56.101:24007/v1/volumes --data @volcreate.json -H 'Content-Type: application/json'
 ```
 
-### Start the volume
+Send the volume create request using glustercli:
+
+    $ ./glustercli volume create testvol <uuid1>:/export/brick1/data <uuid2>:/export/brick2/data --force
```
Comment (prashanthpai) [inline doc/quick-start-user-guide.md]: Please remove `--force`

File: doc/quick-start-user-guide.md [hunk 2]
```diff
@@ -105,6 +105,10 @@ Send a HTTP request to `node1` to add `node2` as peer:
 $ curl -X POST http://192.168.56.101:24007/v1/peers --data @addpeer.json -H 'Content-Type: application/json'
 ```
 
+or using glustercli:
+
+    $ ./glustercli peer probe 192.168.56.102
```
Comment (prashanthpai) [inline doc/quick-start-user-guide.md]: Please use `glustercli` and not `./glustercli` through-out the doc.

File: doc/quick-start-user-guide.md (hunk 1 above)
Comment (prashanthpai) [reply doc/quick-start-user-guide.md]: This example does not match the curl equivalent example (bricks).

File: doc/quick-start-user-guide.md [hunk 3]
```diff
@@ -149,15 +157,24 @@ Send the volume create request to create a 2x2 distributed-replicate volume:
...
+## Start the volume
+
+Send the volume start request:
 
 ```sh
 $ curl -X POST http://192.168.56.101:24007/v1/volumes/testvol/start
 ```
+ or using glustercli:
+
+     $ ./glustercli volume start testvol --force
```
Comment (prashanthpai) [inline doc/quick-start-user-guide.md]: Please remove `--force`

File: doc/quick-start-user-guide.md (hunk 1 above)
Comment (devyanikota) [reply doc/quick-start-user-guide.md]: made changes assuming that the number of bricks was the issue here?
also added the replica flag as in the curl example.

**Pull request merged**